<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>应用程序名</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>in0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_in0"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ID,0返回代表此消息，其他返回值in1一致</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>in1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_in1"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>应用程序名图标名称</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>in2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_in2"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>通知信息概要</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>in3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_in3"/>
      </item>
     </layout>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>通知信息主体</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>in4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_in4"/>
      </item>
     </layout>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>按钮点击后启动的外部应用程序和参数，格式为：程序路径，参数1,参数2。，参数3 .... 参数之间使用逗号隔开</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>in5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_in5"/>
      </item>
     </layout>
    </item>
    <item row="13" column="0" colspan="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>添加按钮，ID和名称&quot;</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>in6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_in6"/>
      </item>
     </layout>
    </item>
    <item row="15" column="0" colspan="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>多长时间超时过期，桌面顶部弹窗多久消失，0为永不消失，-1为5s，&gt;0为指定时间单位ms</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>in7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_in7"/>
      </item>
     </layout>
    </item>
    <item row="17" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="sendBtn">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
